ethereum/retesteth/src/main/java/org/hyperledger/besu/ethereum/retesteth/RetestethContext.java

```diff
@@ -270,7 +270,7 @@ private boolean buildContext(
             syncState,
             transactionPoolConfiguration,
             new BlobCache(),
-            MiningParameters.newDefault());
+            miningParameters);
```

Comment:
Not related to this PR, but IDK if retesteth is a thing anymore.  We should find out if EF testing is using it.  It would be nice to get rid of some more unused features.

Reply:
removed by #7833